src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarArtists.tsx

```diff
@@ -10,7 +10,7 @@ export interface ArtistsProps {
   artwork: ArtworkSidebarArtists_artwork
 }
 
-type Artist = ArtworkSidebarArtists_artwork["artists"][0]
+export type Artist = ArtworkSidebarArtists_artwork["artists"][0]
```

Comment:
Why s this being exported? I don’t see it being used anywhere, but also these types should generally stay local to the module that they are used in.

Reply:
How was it complaining?

Reply:
It was red underlined below and telling that we are using the type that is not exported. @alloy

Reply:
btw... The actual warn is: `[ts] Parameter 'artist' of public method from exported class has or is using private name 'Artist'`. This actually makes sense since some functions on this component take Artist as a parameter.

Reply:
Ohh, I see. Hmm, maybe we should mark the method as private, because it isn’t even meant to be used outside of the component.